### PR TITLE
pin back cloudflare

### PR DESCRIPTION
**Pull request by bmw**
this is the first and easiest step of https://github.com/certbot/certbot/issues/9938

### CODE REVIEW

File: certbot-dns-cloudflare/setup.py [hunk 1]
```diff
@@ -7,7 +7,9 @@
 version = '2.11.0.dev0'
 
 install_requires = [
-    'cloudflare>=1.5.1',
+    # for now, do not upgrade to cloudflare>=2.20 to avoid deprecation warnings and the breaking
```
Comment (zoracon) [inline certbot-dns-cloudflare/setup.py:10]: Is there an expected time frame where we can upgrade this or is this considered indefinite for now?

Comment (bmw) [reply certbot-dns-cloudflare/setup.py:10]: i think it's indefinite for now, but that also might change pretty soon if that makes sense? according to [their announcement](https://github.com/cloudflare/python-cloudflare/discussions/191#discussion-6462999) `cloudflare` 3.0 was supposed to be released last friday but it's still not out yet

as of today, the latest version of the cloudflare package prints warning messages to stderr on every run and their advice is to pin back the dependency

there's more links and discussion about this in the github issue https://github.com/certbot/certbot/issues/9938

Comment (zoracon) [reply certbot-dns-cloudflare/setup.py:10]: Okay, we can just keep watch out for future releases

**Review by zoracon — APPROVED** [review]

**Comment by bmw** [comment]
thanks!

since it sounds to me like [support for the 2.19.x branch will continue indefinitely right now](https://github.com/certbot/certbot/issues/9938#issuecomment-2105745826) and if there's a problem i think users/dependabot will show up to complain, i'm personally not too worried about this. if we wanted to keep an eye on releases though, i'd recommend tracking https://github.com/certbot/certbot/issues/9938 or something like it in a milestone, spreadsheet, etc.

**Pull request merged**